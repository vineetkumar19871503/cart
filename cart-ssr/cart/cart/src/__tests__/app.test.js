import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../components/Home/test';

describe('Welcome (Snapshot)', () => {
  it('Welcome renders hello world', () => {
    const component = renderer.create(<Welcome />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
