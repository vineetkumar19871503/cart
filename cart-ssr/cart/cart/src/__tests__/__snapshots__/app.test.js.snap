// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome (Snapshot) Welcome renders hello world 1`] = `
<div>
  This is testing
</div>
`;
